app/containers/Organisations/Facility.tsx

```diff
@@ -44,7 +44,11 @@ export const FacilityComponent: React.FunctionComponent<Props> = ({
   const applyButton = () => {
     if (!applicationId) {
       return (
-        <Button variant="primary" onClick={startApplication}>
+        <Button
+          variant="primary"
+          data-action="apply"
+          onClick={startApplication}
```

Comment:
style: an `onEvent` should always be bound to a `handleEvent` callback

Reply:
I only added `data-action` attribute here. Im happy to refactor onClick event id necessary.

Reply:
Removed `data-test` attr.